.kuzzlerc

```diff
@@ -66,6 +104,7 @@
         "_canDelete": true,
         "collections": {
           "_canCreate": true,
+          "_canDelete": true,
```

Comment:
not needed here.
the _canDelete role is affected by collection (or under the wildcard - see below)